CHANGELOG.md

```diff
@@ -1,8 +1,45 @@
-# Table of Contents
+# Changelog
 
-- [v0.1.0](#v010)
+## Table of Contents
 
+- [v0.1.0-rc2](#v010-rc2)
+- [v0.1.0-rc1](#v010-rc1)
 
-## v0.1.0
+## v0.1.0-rc2
 
-TBD
+API Version: v1alpha-rc2
+
+### API changes since v0.1.0-rc1
+#### GatewayClass
+- A recommendation to set a `gateway-exists-finalizer.networking.x-k8s.io`
+  finalizer on GatewayClass has been added.
+- `allowedGatewayNamespaces` has been removed from GatewayClass in favor of
+  implementations with policy agents like Gatekeeper.
+
+#### Gateway
+- Fields in `listeners.routes` have been renamed:
+  - `routes.routeSelector` -> `routes.selector`
+  - `routes.routeNamespaces`-> `routes.namespaces`
+- `clientCertificateRef` has been removed from BackendPolicy.
+- In Listeners, `routes.namespaces` now defaults to `{from: "Same"}`.
+- In Listeners, support has been added for specifying custom, domain prefixed
+  protocols.
+- A new `UnsupportedAddress` condition has been added to Listeners to indicate
+  that a requested address is not supported.
+
```

Comment:
346883b has landed and needs to be documented here.

Reply:
Added, thanks!